doc/filters/nl2br.rst

```diff
@@ -2,16 +2,17 @@
 =========
 
 .. versionadded:: 1.5
+
     The ``nl2br`` filter was added in Twig 1.5.
 
 The ``nl2br`` filter inserts HTML line breaks before all newlines in a string:
 
-.. code-block:: twig
+.. code-block:: html+twig
 
     {{ "I like Twig.\nYou will like it too."|nl2br }}
     {# outputs
 
-        I like Twig.<br />
+        I like Twig.<br/>
```

Comment:
I would even remove the / (`<br>`), should be applied everywhere if done.

Reply:
I wouldn't do it. this must render what `|nl2br` renders.

Reply:
Right, so this should be reverted

Reply:
Done